igvc_gazebo/nodes/ground_truth/main.cpp

```diff
@@ -81,29 +88,32 @@ void utm_callback(const ros::TimerEvent& event, const tf::Transform& odom_to_utm
     {
       tf_listener.lookupTransform("odom", "utm", ros::Time(0), transform);
     }
-    catch (const tf::TransformException& ex)
+    catch (const tf::TransformException &ex)
     {
       found = false;
     }
 
     if (found && transform.getRotation() != odom_to_utm.getRotation() &&
         transform.getOrigin() != odom_to_utm.getOrigin())
     {
-      ROS_WARN_STREAM("Anther odom -> utm tf broadcast detected. Disabling ground_truth odom -> utm tf broadcast.");
+      ROS_WARN_STREAM("Anther odom -> utm tf broadcast detected. Disabling "
+                      "ground_truth odom -> utm tf broadcast.");
       enabled = false;
       return;
     }
     br.sendTransform(tf::StampedTransform(odom_to_utm, ros::Time(0), "odom", "utm"));
   }
 }
 
-int main(int argc, char** argv)
+int main(int argc, char **argv)
 {
   ros::init(argc, argv, "ground_truth_republisher");
 
   ros::NodeHandle nh;
   ros::NodeHandle pNh("~");
 
+  nh.getParam("/noise", noise);
+
```

Comment:
You should use the `assertions::parm` helper function to make sure that the node errors out if this is not set.